server/priv/repo/migrations/20220603231003_add_quoted_regtypes_support.exs

```diff
@@ -0,0 +1,291 @@
+defmodule Realtime.RLS.Repo.Migrations.AddQuotedRegtypesSupport do
+  use Ecto.Migration
+
+  def change do
+    execute("
+      alter type realtime.wal_column
+      add attribute type_oid oid cascade,
+      drop attribute if exists type cascade,
+      add attribute type_name text cascade;
```

Comment:
This adjustment to the type changes the attribute order of the `realtime.wal_column` type

it is expected to be 
```sql
create type realtime.wal_column as (
    name text,
    type_name text,
    type_oid oid,
    value jsonb,
    is_pkey boolean,
    is_selectable boolean
);
```

but adding attributes to a type pushes them to the end, so we're getting
```sql
create type realtime.wal_column as (
    name text,
    value jsonb,
    is_pkey boolean,
    is_selectable boolean
    type_oid oid,
    type_name text
);
```

The error message you're receiving is on a line that casts a tuple to `realtime.wal_column` where the values are provided by position (vs name) so I think that will resolve the error